target_postgres/tests/test_standard_target.py

```diff
@@ -347,8 +353,10 @@ def test_no_primary_keys(postgres_target):
     engine = create_engine(postgres_target)
     table_name = "test_no_pk"
     full_table_name = postgres_target.config["default_target_schema"] + "." + table_name
-    with engine.connect() as connection:
-        result = connection.execute(f"DROP TABLE IF EXISTS {full_table_name}")
+    with engine.connect() as connection, connection.begin():
```

Comment:
SQLA 2.0 no longer defaults to auto-commit, so any statement that makes changes to the database (DDL or instert/update/delete) needs to have a begin() beforehand. As with the text() changes, just replicating the behavior of the existing tests.